ts/src/index.ts

```diff
@@ -143,15 +143,12 @@ export class AutoPush {
                 },
               });
         };
-        stream.pushStream(
-            {':path': asset},
-            // Node 9.4.0 changed the callback function signature, hence casting
-            ((err: Error, pushStream: http2.ServerHttp2Stream): void => {
-              if (err) {
-                return reject(err);
-              }
-              pushFile(pushStream);
-            }) as Function as PushStreamCallback);
+        stream.pushStream({':path': asset}, (err, pushStream): void => {
```

Comment:
Not to be nitpicky, but guessing you could drop the `: void` here as well.

Reply:
Personally I prefer being explicit about the return type :)

Reply:
I was confused. Since the types (including the return type) is defined in the `pushStream()` definition, I can delete the return type here. Done. Thanks!